tests/e2e/replication_experimental/multitenancy.py

```diff
@@ -985,7 +1155,29 @@ def test_multitenancy_drop_and_recreate_while_replica_using(connection):
     # 6/ Validate that the transaction is still active and working and that the replica2 is not pointing to anything
 
     # 0/
+    data_directory = tempfile.TemporaryDirectory()
+    MEMGRAPH_INSTANCES_DESCRIPTION = create_memgraph_instances_with_role_recovery(data_directory.name)
     interactive_mg_runner.start_all(MEMGRAPH_INSTANCES_DESCRIPTION)
+
+    replica_1_cursor = connection(BOLT_PORTS["replica_1"], "replica_1").cursor()
+    execute_and_fetch_all(
+        replica_1_cursor, f"SET REPLICATION ROLE TO REPLICA WITH PORT {REPLICATION_PORTS['replica_1']};"
+    )
+
+    replica_2_cursor = connection(BOLT_PORTS["replica_2"], "replica_2").cursor()
+    execute_and_fetch_all(
+        replica_2_cursor, f"SET REPLICATION ROLE TO REPLICA WITH PORT {REPLICATION_PORTS['replica_2']};"
+    )
+
+    main_cursor = connection(BOLT_PORTS["main"], "main").cursor()
+
+    execute_and_fetch_all(
+        main_cursor, f"REGISTER REPLICA replica_1 SYNC TO '127.0.0.1:{REPLICATION_PORTS['replica_1']}';"
+    )
+    execute_and_fetch_all(
+        main_cursor, f"REGISTER REPLICA replica_2 ASYNC TO '127.0.0.1:{REPLICATION_PORTS['replica_2']}';"
```

Comment:
Because of the flag, we keep doing this manually. Can you put them in a function?